grabbit/src/main/proto/node.proto

```diff
@@ -9,6 +9,7 @@ option java_outer_classname = "NodeProtos";
 message Node {
     required string name = 1;
     required Properties properties = 2;
+    repeated Node childNode = 3;
```

Comment:
What do people think of calling this "mandatoryChildNode"?  Just to hint that it's not *all* child nodes being sent in a packet